app/models/robot/verification/source_dest_control_beds.rb

```diff
@@ -16,12 +16,12 @@ def layout_data_object(data_object)
   def valid_plate_locations?(params, batch, robot, expected_plate_layout) # rubocop:todo Metrics/MethodLength
     return false unless super
     unless valid_control_plates_on_robot?(
-             params[:control_bed_barcodes],
-             params[:control_plate_barcodes],
-             robot,
-             batch,
-             expected_plate_layout
-           )
+      params[:control_bed_barcodes],
```

Comment:
Do you know which rule this is? I think either syntax is fine so feels a bit overkill to format it one way or the other.